awl/pitchlabel.cpp

```diff
@@ -76,7 +76,7 @@ void PitchLabel::setValue(int val)
       if (_pitchMode)
             s = pitch2string(_value);
       else
-            s.sprintf("%d", _value);
+            s.asprintf("%d", _value);
```

Comment:
It looks like [`asprintf`](https://doc.qt.io/qt-5/qstring.html#asprintf) is static function in `QString` while [`sprintf`](https://doc.qt.io/qt-5/qstring-obsolete.html#sprintf) is a member function which seems to modify the original string. If it is so, these calls will not behave equivalently and `s` will probably remain an empty string. Maybe something like `s = QString::asprintf("%d", _value);` should be used as a replacement instead.

Reply:
OK, done